server/lib/models/channel.go

```diff
@@ -10,10 +16,14 @@ type Channel struct {
 }
 
 func InitChannel() {
-	create_table(Channel{}, "channels", "Id").AddIndex("established", "INDEX", []string{"Established"})
+	t := create_table(Channel{}, "channels", "Id")
+	t.AddIndex("established", "INDEX", []string{"Established"})
+	t.AddIndex("category", "INDEX", []string{"Category"})
+	t.AddIndex("locale", "INDEX", []string{"Locale"})
+	t.AddIndex("watcher", "INDEX", []string{"Watcher"})
 }
 
-const CHANNEL_FETCH_LIMIT = 10
+const CHANNEL_FETCH_LIMIT = 50
```

Comment:
__[gometalinter]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase
* [golint] exported const CHANNEL_FETCH_LIMIT should have comment or be unexported